src/pkg/packager/deploy.go

```diff
@@ -305,7 +305,13 @@ func (p *Packager) getUpdatedValueTemplate(component types.ZarfComponent) (value
 
 	state, err := p.cluster.LoadZarfState()
 
-	// If no distro the zarf secret did not load properly
+	// YOLO mode but no state found so just fake it this run
+	if p.cfg.Pkg.Metadata.YOLO && state.Distro == "" {
+		state.Distro = "YOLO"
+		err = nil
+	}
+
+	// If there is no state, or the state is invalid, return early
 	if err != nil || state.Distro == "" {
```

Comment:
I know you don't like `else if`, but I wanted to throw in my 2cents that I think this should be one.

Imo it's too easily for someone to accidentally throw code in-between this line and where the err was actually generated.

Reply:
Thoughts? 
<img width="667" alt="Screenshot 2022-12-15 at 3 02 10 PM" src="https://user-images.githubusercontent.com/882485/207966401-33d037a0-88b2-44c0-b774-9fee4de9f1df.png">

Reply:
Or maybe even simpler
<img width="573" alt="Screenshot 2022-12-15 at 3 03 26 PM" src="https://user-images.githubusercontent.com/882485/207966597-75f1f10b-153d-4a1c-8546-6ea6a581a335.png">

Reply:
hmmm well that could maybe still have `err = nil` and `state.Distro = ""`, dang

Reply:
This should cover those different edge cases
<img width="945" alt="Screenshot 2022-12-15 at 3 15 23 PM" src="https://user-images.githubusercontent.com/882485/207968650-cbd9b6ec-b28b-40a9-9fa2-2e156e72111a.png">